lib/ultimate_turbo_modal/base.rb

```diff
@@ -187,7 +192,7 @@ def div_title
       if @title_block.present?
         render @title_block
       else
-        h3(id: "modal-title-h", class: self.class::DIV_TITLE_H_CLASSES) { @title }
+        send(@title_tag, id: "modal-title-h", class: self.class::DIV_TITLE_H_CLASSES) { @title }
```

Comment:
I suggest we leave this as it was before. If the user wants to use a different tag, they can use `title_block`